### DISCUSSION TITLE
Add attachement with note-mark

### DISCUSSION THREAD

**Discussion by hari-bo** [enchant97/note-mark#192] [Ideas]
I've got a lot of notes with attachment like an image/pdf. It would be great if it can be added within note-mark as well instead off an external location. So note-mark isn't dependent for external resources wich could be deleted in the future.
If i look at the memos setup, you can add an attachement wich is placed in an seperate resource folder and within the post it's linked.

**Comment by enchant97** [comment]
In Note Mark you can already upload assets to a note. See the below video:

https://github.com/enchant97/note-mark/assets/25802519/5e0682a4-2b78-4b60-9b60-19ff27426326

**Reply by hari-bo** [reply]
Thanks, totally missed this feature. In the editor mode i just use the default menu and there's an add image button and it shows only the option alt text and URL instead of uploading.
Maybe it is usefull to add this button in the editor as well instead of the other menu?

**Comment by hari-bo** [comment]
It is supported, but would be nice to have to have this option in another menu as mentioned earlier.
